MrtCore/wpf_packaged_app/wpf_packaged_app/wpf_packaged_app.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <Platforms>x86;x64</Platforms>
+    <RuntimeIdentifiers>win10-x86;win10-x64</RuntimeIdentifiers>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.18362.16" />
+    <FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.18362.16" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ProjectReunion" Version="0.5.6" />
+    <PackageReference Include="Microsoft.ProjectReunion.Foundation" Version="0.5.6" />
+  </ItemGroup>
+
+  <!-- Not needed with v0.8 or later of Project Reunion. -->

```

Comment:
><!-- Not needed with v0.8 or later of Project Reunion. --> [](http://example.com/codeflow?start=2&length=58)

i'd rather not have this comment. The explicit include is preferred.

Reply:
Why is it preferred?

Reply:
if you add a new resw file from VS, does it add an entry here?

Reply:
It doesn't

Reply:
then it's not built either. Once you set a build action, it will show up in csproj.

Reply:
We know from prior testing that it is built. That's captured in the Note in the MRT Core docs page.

Reply:
i just tried, and it's not.

Reply:
The behavior between 0.5.0 and 0.5.6 is different (I just tested). The behavior captured in the doc is for 0.5.0.
With 0.5.6, the Build Action is no longer being set "automatically".
Anyway, I'm OK with getting rid of this comment. I'll do it in the next iteration.